RAKE/RAKE.py

```diff
@@ -4,8 +4,11 @@
 # Automatic keyword extraction from indi-vidual documents.
 # In M. W. Berry and J. Kogan (Eds.), Text Mining: Applications and Theory.unknown: John Wiley and Sons, Ltd.
 
+from __future__ import absolute_import
 import re
 import operator
+import six
+from six.moves import range
```

Comment:
Normally I'd agree, but I think for this case it's safe, six is very common in production for 2/3 maintainability. Since the function `six.moves.range` is so similar to the builtin `range` (and the same in python3), and its effect will be scoped to this module (unless explicitly imported), I think the risk of error arising from its use is negligible.